### PR TITLE
Testing Sonar Scanner tool.

### PR DESCRIPTION
**Pull request by AlexisErazoGlobant**

### CODE REVIEW

File: pipeline.yml [hunk 1]
```diff
@@ -16,18 +16,25 @@ schedule :
...
     scripts:
-      - "dotnet build --configuration Release IronPython3Extension.sln"
+      - dotnet build --configuration Release IronPython3Extension.sln
 
 ci_test:
   tests:    
     -
       scripts:
-        - "dotnet test --no-build --configuration Release IronPython3Extension.sln --filter \"TestCategory!=Failure\" --logger:junit"   
+        - dotnet test --no-build --configuration Release IronPython3Extension.sln --filter \"TestCategory!=Failure\" --logger:junit  --logger:trx" -f xml -o "coverage.xml"
```
Comment (sm6srw) [inline pipeline.yml]: The " after trx looks wrong to me.

Comment (sm6srw) [reply pipeline.yml]: And there is no need to escape " for the filters now.

Comment (sm6srw) [reply pipeline.yml]: Oops missed the cover tool. We want to follow this pattern:
`dotnet coverage collect "dotnet test --no-build --configuration Release TestingSolution.sln --logger:junit --logger:trx" -f xml -o "coverage.xml"`
so the escape is still needed for the filters, etc.

Comment (AlexisErazoGlobant) [reply pipeline.yml]: Ups, it was my bad, done!

File: pipeline.yml [hunk 2]
```diff
@@ -16,18 +16,25 @@ schedule :
...
     scripts:
-      - "dotnet build --configuration Release IronPython3Extension.sln"
+      - dotnet build --configuration Release IronPython3Extension.sln
 
 ci_test:
   tests:    
     -
       scripts:
-        - "dotnet test --no-build --configuration Release IronPython3Extension.sln --filter \"TestCategory!=Failure\" --logger:junit"   
+        - dotnet coverage collect "dotnet test --no-build --configuration Release IronPython3Extension.sln --filter TestCategory!=Failure --logger:junit  --logger:trx" -f xml -o "coverage.xml"
```
Comment (sm6srw) [inline pipeline.yml]: You need add back \\" for the filters again. Sorry about that.

Comment (AlexisErazoGlobant) [reply pipeline.yml]: That's ok, done!

**Review by sm6srw — APPROVED** [review]

**Pull request merged**